one_fm/api/doc_methods/shift_request.py

```diff
@@ -37,9 +42,25 @@ def on_update(doc, event):
         workflow_approve_reject(doc, [get_employee_user_id(doc.employee)])
 
     if doc.workflow_state == 'Draft':
+        assign_approver(doc, doc.approver)

```

Comment:
Assign approver by Assignment rule, discuss the same with BA

Reply:
It is using the assignment Rule.